src/Sylius/Behat/Context/Setup/CustomerContext.php

```diff
@@ -80,7 +80,7 @@ public function theStoreHasCustomerWithNameAndEmail($name, $email)
     }
 
     /**
-     * @Given the store has customer :email
+     * @Given the store (also) has customer :email
```

Comment:
@pamil @lchrusciel you two resolve this one, i'm :+1: for the original, which also happens to be almost same as what Lukasz's suggested. Not having the possibility for steps like 
`store  has` (with double spaces), is the way to go imo.